src/BaselineUtils.cc

```diff
@@ -883,26 +976,24 @@ symmetrizeFootprint(
     det::Footprint const& foot,
     int cx, int cy) {
 
-    typedef typename det::Footprint::SpanList SpanList;
-
-    PTR(det::Footprint) sfoot(new det::Footprint(foot.getPeaks().getSchema()));
-    const SpanList spans = foot.getSpans();
-    assert(foot.isNormalized());
+    auto sfoot = std::make_shared<det::Footprint>();
+    sfoot->setPeakSchema(foot.getPeaks().getSchema());
```

Comment:
It can, but only if it is also initialized with a SpanSet at the same time. Which I could have done with an empty SpanSet, I am not sure why I didn't, probably because this seemed more readable [make an empty Footprint, set the schema]. The desire was to keep down the number of constructors, but one that only takes a schema may be worth looking into, I will make a note.